src/test/java/org/broadinstitute/hellbender/IntelDeflaterIntegrationTest.java

```diff
@@ -42,7 +42,7 @@ public void testIntelDeflaterIsAvailable(){
             throw new SkipException("IntelDeflater not available for this architecture");
```

Comment:
The `INPUT_FILE` declared above does not exist (`src/test/resources/CEUTrio.HiSeq.WGS.b37.NA12878.20.21.bam`), though there is a file with that name in `src/test/resources/large`